### DISCUSSION TITLE
Gauge with Python and Headless Chrome (Chromium)

### DISCUSSION THREAD

**Discussion by irynahryshanovich** [getgauge/gauge#2408] [General]
Hello,
Can we use the Gauge framework with Headless Chrome if our tests are written in Python?
I could not find information about it.

I found only one article, but it didn't mention any settings for Python.
https://medium.com/getgauge/testing-with-headless-chrome-9698cf56fdf9

Thank you for your response.
Iryna

**Comment by zabil** [comment]
Yes, you can use the Gauge framework with Headless Chrome for tests written in Python. If you are using Selenium just set it to run on headless mode.

**Comment by haroon-sheikh** [comment]
@irynahryshanovich if it helps, you can use this example project https://github.com/getgauge-examples/python-selenium

**Reply by irynahryshanovich** [reply]
Thank you!
